components/director/internal/open_resource_discovery/client.go

```diff
@@ -117,7 +114,7 @@ func (c *client) fetchConfig(ctx context.Context, app *model.Application, webhoo
 	}
 
 	var resp *http.Response
-	if _, secured := c.securedApplicationTypes[app.Type]; secured {
+	if webhook.Auth != nil {
```

Comment:
Here we check the webhook.Auth if it's nil so in the `httputil.GetRequestWithCredentials` we can remove the first condition check which is the same? - https://github.com/kyma-incubator/compass/blob/main/components/director/pkg/http/request.go#L16

Reply:
Yes, but it's possible for someone to call `httputil.GetRequestWithCredentials` without checking it and so I think it's a good idea to leave it there as well